app/controllers/forgot-password.js

```diff
@@ -0,0 +1,35 @@
+import Controller from '@ember/controller';
+import { action } from "@ember/object";
+import { tracked } from "@glimmer/tracking";
+import config from 'point-maker-web/config/environment';
+
+export default class ForgotPasswordController extends Controller {
+  @tracked email;
+  @tracked error;
+  @tracked success;
+
+  @action
+  onChange(event) {
+    this.set(event.target.name, event.target.value)
+  }
+  
+  @action
+  async submit(event) {
+    event.preventDefault();
+
+    const postData = {user:{email: this.get('email')}};
+    const path = `${config.api}/${config.namespace}/users/password`;
+    try {
+      await window.$.ajax({
```

Comment:
@bhiseweb let's remove jQuery and do this in the ember way. jQuery usage is deprecated in ember now.

Reply:
okay